db/db_tunables.h

```diff
@@ -1821,5 +1821,10 @@ REGISTER_TUNABLE("debug_queuedb",
                  "(Default: off)",
                  TUNABLE_BOOLEAN, &gbl_debug_queuedb, EXPERIMENTAL, NULL, NULL,
                  NULL, NULL);
-
+REGISTER_TUNABLE("scconvert_finish_delay",
+                 "Delay returning from convert_record when a stripe finishes. "
+                 "This would create a scenario where scgenids are on the right "
+                 "of any new genids to reproduce a vutf8 schema change bug. ",
+                 TUNABLE_BOOLEAN, &gbl_scconvert_finish_delay,
+                 EXPERIMENTAL | INTERNAL, NULL, NULL, NULL, NULL);
```

Comment:
[Optional] I'd prefix this with 'debug_' to make it explicit that this tunable is intended for debugging/testing.  Also, suggest you to make it of INTEGER type, so that the sleep duration in the test can later be adjusted.

Reply:
Thanks for the advice. I've made it a debug switch. I'm not quite in favor of making it an integer as sleep duration is unlikely to change. A hardcoded value would suffice.